backend/src/main/java/com/bakdata/conquery/models/common/Range.java

```diff
@@ -32,18 +31,18 @@ public Range(T min, T max){
 	@Override
 	public String toString() {
 		if (isExactly()) {
```

Comment:
String format ist super langsam, deswegen hab ich das in von Hand gemacht.

Reply:
Wird das denn oft aufgerufen?

Reply:
Ne, war aber drüber gestolpert, deswegen hab ich es angepasst. Stört es dich?